### DISCUSSION TITLE
ET SCAN UPnP SUBSCRIBE Inbound - Possible CallStranger Scan (CVE-2020-12695)

### DISCUSSION THREAD

**Discussion by sblanch** [Security-Onion-Solutions/securityonion#6923] [Unsupported Versions] [answered]
This alert is configured as:
alert http **$EXTERNAL_NET any -> $HOME_NET** any (msg:"ET SCAN UPnP SUBSCRIBE Inbound - Possible CallStranger Scan (CVE-2020-12695)"; flow:established,to_server; http.method; content:"SUBSCRIBE"; nocase; http.header; content:"CALLBACK|3a 20|"; fast_pattern; content:"NT|3a 20|"; reference:cve,2020-12695; reference:url,kb.cert.org/vuls/id/339275; classtype:attempted-recon; sid:2030272; rev:2; metadata:attack_target Client_Endpoint, created_at 2020_06_09, deployment Perimeter, former_category SCAN, signature_severity Informational, updated_at 2020_06_09;)

but both devices (192.168.88.53, 192.168.88.44) are on my home network (192.168.88.0/24) so not sure why it triggered.

My global.sls has the following:
 hnmanager: '192.168.88.0/24,192.168.101.0/24,192.168.102.0/24,192.168.103.0/24'

Is there another location I need to make sure my home network is defined.

This was triggered by my iPhone Sonos App connecting to my Sonos speaker

Any thoughts are appreciated.

**Comment by dougburks** [comment, marked as answer]
`$EXTERNAL_NET` defaults to `any` which includes `$HOME_NET`:
https://docs.securityonion.net/en/2.3/suricata.html?highlight=external_net#configuration
